qt/scientific_interfaces/test/ISISReflectometry/Reduction/ValidatePerThetaDefaultsTest.h

```diff
@@ -132,6 +132,25 @@ class ValidatePerThetaDefaultsTest : public CxxTest::TestSuite {
     TS_ASSERT(result.isError());
     TS_ASSERT_EQUALS(result.assertError(), errorCells);
   }
+
+  void testParseBackgroundProcessingInstructions() {
+    PerThetaDefaultsValidator validator;
+    auto result = validator({"", "", "", "", "", "", "", "", "", "4-7"});
+    TS_ASSERT(result.isValid());
+    TS_ASSERT(result.assertValid()
+                  .backgroundProcessingInstructions()
+                  .is_initialized());
+    TS_ASSERT_EQUALS(
+        result.assertValid().backgroundProcessingInstructions().get(), "4-7");
+  }
+
+  void testParseBackgroundProcessingInstructionsError() {
+    PerThetaDefaultsValidator validator;
+    auto result = validator({"", "", "", "", "", "", "", "", "", "bad"});
+    std::vector<int> errorCells = {9};
```

Comment:
Again, could `9` be replaced with `PerThetaDefaults::BACKGROUND_SPECTRA`?  Obviously this would also require including `PerThetaDefaults.h`